hummingbot/client/command/connect_command.py

```diff
@@ -10,7 +10,7 @@
 if TYPE_CHECKING:
     from hummingbot.client.hummingbot_application import HummingbotApplication
 
-OPTIONS = settings.CEXES.union({"ethereum", "celo"})
+OPTIONS = settings.EXCHANGES.union({"ethereum", "celo"})
```

Comment:
I've realised this was probably intentional :D so I can remove this from the PR if you're not ready to enable DEX's yet.

Reply:
Yep, that's correct. For current DEX exchanges, they are all Ethereum based, so users connect to `ethereum` rather than the exchange. Though, we'll need to change this to make it explicitly clear.

Reply:
yes, should remain as `OPTIONS = settings.CEXES.union({"ethereum", "celo"})` like @Nullably  has explained.